DataProcessing/datax-host/src/main/scala/datax/telemetry/AppInsightLogger.scala

```diff
@@ -37,6 +41,11 @@ object AppInsightLogger extends TelemetryService {
       logger.warn("AI Key is not found, AppInsight Sender is OFF")
   }
 
+  private val bAppenderEnabled = (for {
```

Comment:
what does b stand for in bAppenderEnabled. Please name the val so that is more descriptive.